tips/tests/tips_views_tests.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+from django.utils import timezone
+from django.shortcuts import get_object_or_404
+from tips.models import Tip
+from django.template import TemplateDoesNotExist
+from django.test.client import RequestFactory
+from tips.forms import TipForm
+from tips.views import board
+from django.template.loader import get_template
+
+AUTHOR = 'RICK'
+OTHER_AUTHOR = 'RICKEN'
+CATEGORY = 'Clothing'
+DATE = timezone.now()
+TEXT = 'I BOUGHT A TESLA IN 50$ DISCOUNT IN LEVINSKI MARKET'
+
+FORM_DATA = {'author': OTHER_AUTHOR, 'text': TEXT, 'category': CATEGORY}
```

Comment:
When writing a test we want to check bad data as well. we are checking for edge cases.

Reply:
done